shared/parcel.js

```diff
@@ -20,21 +20,14 @@ export function isParcel(asset) {
   )
 }
 
-export function toParcelObject(
-  parcelsArray,
-  prevParcels = [],
-  normalize = true,
-  redoConnections = false
```

Comment:
If we remove the `redoConnections` we will lose the _feature_ of a confirmed tx of removing a parcel from an Estate

Reply:
That's taken care by this line: https://github.com/decentraland/marketplace/pull/440/files#diff-fa45755e5e2585857eb74278da106dd6R51

Reply:
Yep, with the new commit is fixed 👍